### PR TITLE
CI tweaks

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -230,7 +233,8 @@ jobs:
       - checkout
       - build
       - test
-      - benchmark
+      - benchmark:
+          min_time: "0.01"
```
Comment (axic) [inline circle.yml:237]: What is this doing?

File: codecov.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 codecov:
-  require_ci_to_pass: yes
+  require_ci_to_pass: no
```
Comment (axic) [inline codecov.yml:2]: Why?

File: circle.yml (hunk 1 above)
Comment (chfast) [reply circle.yml:237]: Normally benchmarks run for at least 0.5 s. But in sanitizers run we only want to find bugs in benchmarking code. With this they run only for 10 ms and whole job finishes faster. Except for the ecpairing where single iteration takes ~30 s.

File: codecov.yml (hunk 2 above)
Comment (chfast) [reply codecov.yml]: Nvm, I thought it did something else.

**Pull request merged**